index.html

```diff
@@ -2174,29 +2196,29 @@ <h2>WoT System Components and their Interconnectivity</h2>
 
             <p>On the other hand, <a>Consumers</a> are always implemented by 
                <a>Servients</a>, as they must be able to process the <a>Thing Description</a> (TD) 
-               format and must have a protocol stack that can be configured through 
+               format and must have a protocol stack that can be configured to use a thing through 
```

Comment:
The protocol stack itself needs to be configured. The configuration information is given in the Protocol Binding. Hence, "configured through" or "by" Protocol Binding information.

Reply:
I agree that the protocol stack needs to be configured, but that's still expressed here.
My change explains the purpose of configuring the protocol stack, i.e. "to use a thing".
IMHO the change adds clarity.

Reply:
"configured to use a *T*hing" sounds to me like just giving a target... maybe "configured for a Thing through"?

Reply:
👍